### DISCUSSION TITLE
Way to preserve muscle-memory for C-x C-s when in magit-wip-mode

### DISCUSSION THREAD

**Discussion by dabrahams** [magit/magit#4385] [Support] [answered]
`magit-wip-mode` is awesome, but I keep getting tripped up by the way `C-x C-s' ends up asking me about whether to save other files I've modified. It puts a hiccup in my regular workflow that I normally never expect to see.  I think I'd just be happy for it to save everything that's been added to the repo without asking at that point, but *only* when being in wip-mode is causing the query. Is there a way to configure that? I've tried to find an answer in the docs but came up empty.  Thanks!

**Comment by tarsius** [comment, marked as answer]
> I keep getting tripped up by the way `C-x C-s` ends up asking me about whether to save other files I've modified.

Emacs does that regardless of whether you use `magit-wip-mode` or not. `magit-wip-mode` doesn't even need to trigger buffers to be saved. Instead it kicks in when *something else* saves buffers.

Other part of Magit do though and you probably want `(setq magit-save-repository-buffers 'dontask)`.
